doc/user/CONFIGURATION_FILE.md

```diff
@@ -65,12 +65,14 @@ The following command-line options <b>can only</b> be set in the global block of
 Boolean options that have been turned on in the configuration file can be turned
 off on the command line if needed, eg: `--point-sprites=false`.
 
-The configuration file or directory possible locations depends on your operating system.
-They are considered in the below order and only the first found will be used.
+F3D looks for configuration files in different locations depending on your operating system.
+Existing configuration files are read in order and combined with later entries overriding earlier ones.
```

Comment:
I'm not sure but `earlier` doesn't sound right to me. What about `previous`?

```suggestion
Existing configuration files are read in order and combined with later entries overriding previous ones.
```

Reply:
I dont like previous either.

` with later entries, potentially overriding previously read entry with the same names.` ?

Reply:
ok sounds good